### ISSUE TITLE
IntelliSense breaks for Node.js imports

### ISSUE THREAD

**Issue by stijnherreman** [Microsoft/vscode#53531]
Issue Type: <b>Bug</b>

IntelliSense works for a file with this content:

```js
import { readdirSync } from "fs";

const filePaths = readdirSync("");
```

It also works for a file with this content:

```js
import { readdirSync } from "fs";
import { parse } from "path";

const filePaths = readdirSync("")
  .map(filePath => ({
    filePath: filePath,
    parsedFilePath: parse(filePath)
  }))
  .filter(file => {
    return file.parsedFilePath.ext === ".adoc";
  });
```

But at some point IntelliSense breaks, while I'm adding code. Restarting VS Code sometimes fixes it.

VS Code version: Code 1.24.1 (24f62626b222e9a8313213fb64b10d741a326288, 2018-06-13T17:51:32.889Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz (8 x 2712)|
|GPU Status|2d_canvas: unavailable_software<br>flash_3d: unavailable_software<br>flash_stage3d: unavailable_software<br>flash_stage3d_baseline: unavailable_software<br>gpu_compositing: unavailable_software<br>multiple_raster_threads: unavailable_off<br>native_gpu_memory_buffers: disabled_software<br>rasterization: unavailable_software<br>video_decode: unavailable_software<br>video_encode: unavailable_software<br>vpx_decode: unavailable_software<br>webgl: unavailable_off<br>webgl2: unavailable_off|
|Memory (System)|15.86GB (8.15GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (9)</summary>

Extension|Author (truncated)|Version
---|---|---
ng-template|Ang|0.1.9
gitlens|eam|8.4.1
EditorConfig|Edi|0.12.4
tslint|eg2|1.0.33
prettier-vscode|esb|1.5.0
asciidoctor-vscode|joa|0.12.0
team|ms-|1.136.0
vscode-icons|rob|7.24.0
sass-indented|rob|1.4.9

</details>

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code build](https://code.visualstudio.com/insiders/) with all extensions disabled?

**Comment by stijnherreman** [comment]
At first sight it doesn't reproduce in the latest build. I'm not a daily user though, so I can't tell if it never reproduces. Are there fixes to IntelliSense in the pipeline for next release?

**Comment by mjbvz** [comment]
Closing as not-reproducable / we have not heard that the issue was not fixed by VS code 1.25+
